### PR TITLE
Refactor imu demo

### PR DESCRIPTION
**Pull request by caguero**
Refactors the `imu` demo leveraging the latest improvements in `ros_gz`. 

* I replaced the previous launch file from Python to XML.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: ros_gz_sim_demos/launch/imu.launch.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <gz_server
+    world_sdf_file="sensors.sdf"
+    use_composition="True"
+    create_own_container="True" />
+  <ros_gz_bridge
+    bridge_name="ros_gz_bridge"
+    config_file="$(find-pkg-share ros_gz_sim_demos)/config/imu.yaml"
+    use_composition="True" />
+  <node pkg="rqt_topic" exec="rqt_topic" args="-t" />
+  <!-- FIXME: Add once there's an IMU display for RViz2 -->
```
Comment (Amronos) [inline ros_gz_sim_demos/launch/imu.launch.xml]: The [rviz_imu_plugin](https://index.ros.org/p/rviz_imu_plugin/#rolling) package does provide this functionality.

**Review by ahcorde — COMMENTED** [review]
I can't see the value in rqt_topic

**Comment by caguero** [comment]
> I can't see the value in rqt_topic

I can see the values:

Screenshot from 2024-12-13 15-35-45

**Review by ahcorde — APPROVED** [review]
LGTM, This other PR https://github.com/gazebosim/ros_gz/pull/675 should fix the RViz visualization

**Pull request merged**